src/main/app/forms/models/representativeDetails.ts

```diff
@@ -0,0 +1,30 @@
+import { Address } from 'forms/models/address'
+import { ContactDetails } from 'forms/models/contactDetails'
+import { OrganisationName } from 'forms/models/organisationName'
+import { FeeAccount } from 'forms/models/feeAccount'
+import * as express from 'express'
+import CookieProperties from 'common/cookieProperties'
+
+export class RepresentativeDetails {
+  organisationName: OrganisationName = new OrganisationName()
+  address: Address = new Address()
+  contactDetails: ContactDetails = new ContactDetails()
+  feeAccount: FeeAccount = new FeeAccount()
+  cookieName: string = ''
```

Comment:
why initialise it to empty string?

Reply:
it was having issues with accessing undefined. I'll double check to see if it is still needed